packages/core/lib/services/credential/service.js

```diff
@@ -0,0 +1,36 @@
+import {serviceName, validation} from './config';
+import VerifiableCredential from '@docknetwork/sdk/verifiable-credential';
+import {getKeypairFromDoc} from '@docknetwork/wallet/methods/keypairs';
+import {getSuiteFromKeyDoc} from '@docknetwork/sdk/utils/vc/helpers';
+
+class CredentialService {
+  constructor() {
+    this.name = serviceName;
+  }
+  rpcMethods = [
+    CredentialService.prototype.generateCredential,
+    CredentialService.prototype.signCredential,
+  ];
+  generateCredential() {
+    const vc = new VerifiableCredential();
+    vc.addType('DockAuthCredential');
+    return vc;
+  }
+  async signCredential(params) {
```

Comment:
do these `params` need any validation?

Reply:
Yes, that is what validation.signCredential(params); is for.